cfpb/capital-framework
# Docs: Update for capital-framework 5.0.4

Most notably, this fixes a small issue in the cf-notifications docs where the example for a warning notification was using the wrong icon.

## Testing

1. Pull branch
2. `npm start`
3. Visit http://localhost:3000/components/cf-notifications/#warning-notification
4. Observe the exclamation mark icon in the warning notification

## Checklist
